amqp/src/main/scala/akka/stream/alpakka/amqp/AmqpSinkStage.scala

```diff
@@ -60,18 +59,7 @@ final class AmqpSinkStage(settings: AmqpSinkSettings)
       private val exchange = settings.exchange.getOrElse("")
       private val routingKey = settings.routingKey.getOrElse("")
 
-      override def whenConnected(): Unit = {
```

Comment:
This is 100% unnecessary since we are already creating a shutdown connector on the `ConnectorLogic` doing exactly the same.